server/src/weaverbird/backends/mongo_translator/steps/sort.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+
+from weaverbird.backends.mongo_translator.steps.types import MongoStep
+from weaverbird.pipeline.steps import SortStep
+
+
+def translate_sort(step: SortStep) -> List[MongoStep]:
+    sort_mongo = {}
```

Comment:
It seemed to me, that this function was a bit refactored in another step ? But this is ok still.

Reply:
Oh it was missed in the rebase, it should not appear on this PR